source/style-guide/style/text-formatting.txt

```diff
@@ -4,603 +4,1636 @@
+
+               If your comment to an acknowledgement is too long, the
+               API returns:
+
+               ``Acknowledgement comment too long. It must not exceed
+               <number> characters.``
+
+          - If your comment to an acknowledgement is too long, the API
+            returns:
+
+            ``Acknowledgement comment too long. It must not exceed
+            <number> characters.``
+
+   Examples, code
+     Format using the ``.. code-block::`` directive with corresponding.
+       `Pygment lexer <https://pygments.org/docs/lexers/>`__
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               .. code-block:: python
+                  :copyable: false
+
+                  db.collection.create_index(
+                      [
+                        (<keyAndIndexTypeSpecification>
+                      ], <options>
+                    )
+
+               .. code-block:: java
+                  :copyable: false
+
+                  collection.createIndex(
+                    <keyAndIndexTypeSpecification>, <options>
+                    )
+
+          -
+            .. code-block:: python
+               :copyable: false
+
+               db.collection.create_index(
+                   [
+                     (<keyAndIndexTypeSpecification>)
+                   ], <options>
+                 )
+
+            .. code-block:: java
+               :copyable: false
+
+               collection.createIndex(
+                 <keyAndIndexTypeSpecification>, <options>
+                 )
+
+   Field names, GUI
+     Format using ``:guilabel:`` role.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               In the :guilabel:`Database Name` field, enter a database
+               name identifier.
+
+          - In the :guilabel:`Database Name` field, enter a database
+            name identifier.
+
+   File names and extensions
+     Format using ``monospace`` text.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               To remove the ``vs_quantum-api.cfg`` file from the
+               ``haproxy.d`` directory and retain it, you can move it
+               to another directory.
+
+          - To remove the ``vs_quantum-api.cfg`` file from the
+            ``haproxy.d`` directory and retain it, you can move it to
+            another directory.
+
+   Flags
+     Format using ``monospace`` text.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               Use the ``-t`` flag to add a time stamp to the results.
```

Comment:
Nitpick: "time stamp" -> "timestamp"

Reply:
It was `time stamp` in the glossary, but I'm changing it to `timestamp` there.